include/irr/asset/filters/CMatchedSizeInOutImageFilterCommon.h

```diff
@@ -55,10 +119,11 @@ class CMatchedSizeInOutImageFilterCommon : public CBasicImageFilterCommon
 						uint32_t		outBaseLayer;
 					};
 				};
-				uint32_t				inMipLevel = 0u;
-				uint32_t				outMipLevel = 0u;
-				const ICPUImage*		inImage = nullptr;
-				ICPUImage*				outImage = nullptr;
+
+				uint32_t				inMipLevel = 0u;		//!< Current handled mipmap level in reference to \binput\b image
+				uint32_t				outMipLevel = 0u;		//!< Current handled mipmap level in reference to \boutput\b image
+				const ICPUImage* inImage = nullptr;				//!< \bInput\b image being a reference for state management, needed to operate on output image's texel buffer
+				ICPUImage* outImage = nullptr;					//!< \bOutput\b image, it's attached texel buffer will be converted according to state data after execute call
```

Comment:
converted values will be written to texel buffer, texel buffer won't be converted, its already in that format, but doesn't have data inside that makes sense

Reply:
adjusted in following commit